src/main/java/net/sf/jabref/event/AddEntryEvent.java

```diff
@@ -0,0 +1,21 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+
+public class AddEntryEvent {
+
+    private BibEntry bibEntry;
+
+    public AddEntryEvent(BibEntry bibEntry) {
+        setBibEntry(bibEntry);
+    }
+
+    public void setBibEntry(BibEntry bibEntry) {
```

Comment:
Is this method necessary? I would remove it and inline the setting in the constructor.

Reason: the event is set in the constructor